<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiningPage</class>
 <widget class="QWidget" name="MiningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digging</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="15" column="2">
    <widget class="QLabel" name="hashLabel">
     <property name="text">
      <string>Hash-Parallel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="usernameLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QComboBox" name="offloadSHA">
     <item>
      <property name="text">
       <string>Offload to 1 thread.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Offload to all threads.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All hashing to GPU.</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QComboBox" name="textureCache">
     <item>
      <property name="text">
       <string>No Cache</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1D Cache</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2D Cache</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="24" column="2">
    <widget class="QPushButton" name="startButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Start Digging</string>
     </property>
    </widget>
   </item>
   <item row="24" column="3">
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="text">
      <string>Debug Logging</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLabel" name="workloadLabel">
     <property name="text">
      <string>Worksize:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="concurrency"/>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="portLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="passwordLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="threadsLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Threads</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QLineEdit" name="kernel"/>
   </item>
   <item row="13" column="4">
    <widget class="QLabel" name="kernelLabel">
     <property name="text">
      <string>Kernel:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLineEdit" name="workload"/>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="cacheLabel">
     <property name="text">
      <string>Texture Cache:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QSpinBox" name="scantimeBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="diggerLabel">
     <property name="text">
      <string>Use Digger:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="threadsBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="4">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="typeBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Internal Dig</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solo Dig</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Herd Dig</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pond Dig</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="portLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>9919</string>
     </property>
     <property name="value" stdset="0">
      <number>9332</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="serverLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string notr="true">fox.doxmon.com:9199</string>
     </property>
     <property name="placeholderText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="usernameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="serverLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1" colspan="4">
    <widget class="QListWidget" name="list">
     <property name="autoScrollMargin">
      <number>100000</number>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Snap</enum>
     </property>
     <property name="batchSize">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="passwordLine">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>foxes</string>
     </property>
     <property name="placeholderText">
      <string comment="foxes" extracomment="foxess"/>
     </property>
    </widget>
   </item>
   <item row="25" column="3" colspan="2">
    <widget class="QLabel" name="mineSpeedLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="scantimeLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Scantime</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QSlider" name="intensity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="4">
    <widget class="QLabel" name="advancedLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;---Advanced Options---&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1" colspan="2">
    <widget class="QLabel" name="shareCount">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="useDigger">
     <item>
      <property name="text">
       <string>Minerd [CPU]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CUDAMiner [nVIDIA GPU]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CGMiner [AMD GPU]</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="concurrencyLabel">
     <property name="text">
      <string>Thread-Concurrency:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="intensityLabel">
     <property name="text">
      <string>Intensity:</string>
     </property>
    </widget>
   </item>
   <item row="15" column="3">
    <widget class="QLabel" name="memoryLabel">
     <property name="text">
      <string>Memory Block:</string>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="QComboBox" name="memoryBlock">
     <item>
      <property name="text">
       <string>Unlimited Blocks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 Block</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="4">
    <widget class="QLabel" name="autotuneLabel">
     <property name="text">
      <string>Autotune</string>
     </property>
    </widget>
   </item>
   <item row="16" column="4">
    <widget class="QComboBox" name="autotune">
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>On</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>typeBox</tabstop>
  <tabstop>serverLine</tabstop>
  <tabstop>portLine</tabstop>
  <tabstop>usernameLine</tabstop>
  <tabstop>passwordLine</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
